framework/include/cppmicroservices/BundleContext.h

```diff
@@ -1069,11 +1070,21 @@ class US_Framework_EXPORT BundleContext
    * which may contain several bundles, i. e. acts as a bundle library.
    *
    * @param location The location of the bundle library to install.
+   * @param bundleManifest the manifest of the bundle at "location". If non-empty
+   *        this will be used without opening the bundle at "location". Otherwise, the bundle will
+   *        be opened and the manifest read from there.
```

Comment:
there is no guidance on how to pass in a manifest which describes multiple statically linked bundles. How and where will this be documented?

Reply:
I can add something to this doxygen comment describing the bundleManifest AnyMap schema. Or is there someplace else that makes more sense?

Reply:
I think adding the documentation at the API level is appropriate here.

Reply:
Perhaps I should just put a link to the RFC in the Doxygen comment to the spot in doc that contains a description of the manifest format?

Reply:
you could. keep in mind that this API level doxygen documentation appears on docs.cppmicroservices.org, e.g. http://docs.cppmicroservices.org/en/latest/framework/doc/api/main/BundleContext.html. If the RFC link is never going to change then it makes sense to add a link.

Reply:
I added a TBD comment to remind us to document the manifest format elsewhere (in an rst file as you suggested).